cpp_linter/cli.py

```diff
@@ -358,6 +362,22 @@ class Args(UserDict):
     help="""Set to ``true`` to prevent Pull Request
 reviews from requesting or approving changes.""",
 )
+_parser_args[("-C", "--delete-comments")] = dict(
+    default="true",
+    type=lambda input: input.lower() == "true",
+    help="""Set to ``true`` to delete existing outdated/unused
+review comments, ``false`` to just set them to resolved.
+
+Defaults to ``%(default)s``.""",
+)
+_parser_args[("-U", "--reuse-comments")] = dict(
+    default="true",
+    type=lambda input: input.lower() == "true",
+    help="""Set to ``true`` to reuse existing review
+comments if nothing has changed instead of making new ones.
+
+Defaults to ``%(default)s``.""",
+)
```

Comment:
The behavior when `--thread-comments` is set to 'update'. Why do we need a separate/new CLI option?